pkg/pgmodel/migrate.go

```diff
@@ -94,21 +94,33 @@ func Migrate(db *sql.DB) error {
 	if err != nil {
 		return err
 	}
+	defer func() {
+		sErr, dErr := m.Close()
+		if err != nil {
+			return
+		}
+		if sErr != nil {
+			err = sErr
+			return
+		}
+		if dErr != nil {
+			err = dErr
+			return
```

Comment:
nit: consider adding some comments to describe the code block, or alternatively finding descriptive variable names..

Reply:
done